ubports-qa

```diff
@@ -118,13 +123,31 @@ def add_list(branch):
         repo_list.write("deb http://repo.ubports.com/ {} main".format(branch))
 
 
+def add_pref(branch):
+    if pref_exists(branch):
+        return
+    with open(get_pref_file(branch), "w+") as pref:
+        # TODO: edit and set own pin priority to allow up/down/side-grade without
+        # removal of repo etc
+        pref.write("""Package: *\n"""
+                   """Pin: release o=UBports,a={}\n"""
+                   """Pin-Priority: 2000\n""".format(branch))
```

Comment:
Triple quotes shouldn't be necessary here. Triple quotes are used to bound multi-line strings:

```python
""" This is the start of a really long string

It continues all the way down here."""
```

Single quotes should work:
```python
>>> ("Package: *\n"
... "Pin: releaseo=UBports,a={}\n"
... "Pin-Priority: 2000\n\n".format("release"))
'Package: *\nPin: releaseo=UBports,a=release\nPin-Priority: 2000\n\n'
```